### ISSUE TITLE
dbmldoc is broken in foliant.

### ISSUE THREAD

**Issue by rubiagatra** [foliant-docs/foliantcontrib.dbmldoc#1]
I follow your instruction for generating DBML using foliant 

```
~/experimental/database-docs 
❯ foliant make site -w mkdocs
Parsing config... Done
Applying preprocessor dbmldoc... Failed to apply preprocessor dbmldoc: format_exception() got an unexpected keyword argument 'etype'
Applying preprocessor mkdocs... Done
Applying preprocessor _unescape... Done
Making site with MkDocs... Done
────────────────────
Result: Database_Docs-2022-11-15.mkdocs
```

I am using Mac

**Comment by rubiagatra** [comment]
Do you have any idea @Vanderhoof ?

**Comment by Vanderhoof** [comment]
@rubiagatra Thanks for the report! I'll take a look. Probably because PyDBML was updated

**Comment by Vanderhoof** [comment]
@rubiagatra my guess was right: the jinja2 template is not compatible with PyDBML 3.0+ version. I've fixed it (not released yet), but my error message was different from yourse. Could you share the debug log? To do it run the foliant build command with --debug key, for example:

```bash
$ foliant make site --with mkdocs --debug
```

I'd like to take a look either at the `.log` file that is generated as the result, or at the console output.

Thanks a lot for your help!

**Comment by Vanderhoof** [comment]
Fixed in 0.3.2
Also updated the default template slightly

**Comment by rubiagatra** [comment]
Thank you! I should close the issue
